src/components/EmailForm/index.tsx

```diff
@@ -3,32 +3,31 @@
 import { yupResolver } from '@hookform/resolvers/yup';
 import { FormProvider, useForm } from 'react-hook-form';
 import { useState } from 'react';
-import cx from 'classnames';
 
+import { postMessage } from '../../api/messages';
 import { Textarea } from '../Textarea';
-import { emailSchema } from './schema';
+import { textSchema } from './schema';
+import { Popup } from '../Popup';
+import { PopupContent } from './PopupContent';
+import { Button } from '../Button';
 
-type EmailBody = {
-  email_body: string;
+type EmailFormProps = {
+  event_id: string;
 };
 
-type EmailFormProps = {
-  eventName: string;
-  mail: string;
+type Text = {
+  text: string;
 };
 
-export const EmailForm: React.FC<EmailFormProps> = ({ eventName, mail }) => {
+export const EmailForm: React.FC<EmailFormProps> = ({ event_id }) => {
+  const [isLoading, setIsLoading] = useState(false);
   const [isModalOpen, setIsModalOpen] = useState(false);
-  const [emailData, setEmailData] = useState({
-    subject: '',
-    mail: '',
-    body: { email_body: '' },
-  });
+  const [status, setStatus] = useState<number | undefined>(undefined);
```

Comment:
Why do we need the explicit undefined? :D 
```suggestion
  const [status, setStatus] = useState<number>();
```

Reply:
Done.